test/compile/common.test.ts

```diff
@@ -95,4 +96,151 @@ describe('Common', function() {
       assert.deepEqual(p.opacity.value, 0.5);
     });
   });
+
+  describe('hasGeoTransform()', function() {
+    describe('should return true iff path mark is specified and encoding.geopath.type is geojson', function() {
```

Comment:
This describe reads funny.  Do you actually want to create multiple `it`s instead?